### PR TITLE
feat(stories): Add basic form fields story

### PR DESCRIPTION
**Pull request by evanpurkhiser**
image

### CODE REVIEW

File: static/app/components/forms/index.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+import RadioField from './fields/radioField';
+import RangeField from './fields/rangeField';
+import SecretField from './fields/secretField';
+import SelectField from './fields/selectField';
+import SentryMemberTeamSelectorField from './fields/sentryMemberTeamSelectorField';
+import SentryProjectSelectorField from './fields/sentryProjectSelectorField';
+import SeparatorField from './fields/separatorField';
+import TextareaField from './fields/textareaField';
+import TextField from './fields/textField';
+import Form from './form';
```
Comment (ryan953) [inline static/app/components/forms/index.stories.tsx:21]: Lets make stories for each input field type on its own. A `<Form>` is not the same as a field and having everything in a giant file will not make it easier for people to find things in the future, or iterate on the examples

Comment (evanpurkhiser) [reply static/app/components/forms/index.stories.tsx:21]: For this I just want to see them all together.

I'm not going to spend more time on this now.

Comment (evanpurkhiser) [reply static/app/components/forms/index.stories.tsx:21]: IMO this is a very good starting point for seeing "what can I do with a form"

Comment (ryan953) [reply static/app/components/forms/index.stories.tsx:21]: I'd rename it `static/app/components/forms/fields/index.stories.tsx` at the very least. It's really not showing anything to do with `<Form>` specifically.

File: static/app/components/forms/index.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+
+export default storyBook(Form, story => {
```
Comment (ryan953) [inline static/app/components/forms/index.stories.tsx:23]: And change this
```suggestion
export default storyBook('Form fields', story => {
```

File: static/app/components/forms/index.stories.tsx (hunk 1 above)
Comment (evanpurkhiser) [reply static/app/components/forms/index.stories.tsx:21]: Let me have it render the form state real quick, then it will be more like a form

Comment (ryan953) [reply static/app/components/forms/index.stories.tsx:21]: The "default" example for `<Form>` probably could just be without the fields, then it'll be easier to see the Submit button gets auto-inserted.

Then another example with fields and state is helpful

Comment (evanpurkhiser) [reply static/app/components/forms/index.stories.tsx:21]: I'll move it over. Will add the form state thing later

File: static/app/components/forms/fields/index.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+import RangeField from './rangeField';
+import SecretField from './secretField';
+import SelectField from './selectField';
+import SentryMemberTeamSelectorField from './sentryMemberTeamSelectorField';
+import SentryProjectSelectorField from './sentryProjectSelectorField';
+import SeparatorField from './separatorField';
+import TextareaField from './textareaField';
+import TextField from './textField';
+
+export default storyBook(Form, story => {
```
Comment (ryan953) [inline static/app/components/forms/fields/index.stories.tsx:24]:
```suggestion
export default storyBook('Form Fields', story => {
```

**Review by ryan953 — APPROVED** [review]

**Pull request merged**